File: problem9/p9.py
def pythagorean_triplet(n):
    for a in range(1, int(n/3)):
        for b in range(a+1, int(n/2)):
            c = int(n) - (a+b)
            if a + b + c == 1000:
                if a**2 + b**2 == c**2:
                    print(a, "+", b, "=", c, "and", a, "^2 +", b, "^2 =", c, "^2")
                    return a*b*c


print(pythagorean_triplet(1000))

File: problem9/p9.java
public class p9 {
	
	public static int pythagorean_triplet(int n) {
		
		int a, b, c;
		for(a=1; a<n/3; a++) {
			for(b=a+1; b<n/2; b++) {
				c = n - (a + b);
				if((a + b + c) == 1000) {
					if(a*a + b*b == c*c) {
						System.out.println(a + " + " + b + " = " + c + " and " + a + "^2 + " + b + "^2 = " + c + "^2");
						return a*b*c;
					}
				}
			}
		}
		
		return -1;
		
	}

	public static void main(String args[]) {
		
		int res = pythagorean_triplet(1000);
		
		if(res != -1) {
			System.out.println(res);
		} else {
			System.out.println("Could not found");
		}
		
	}

}
